src/agent/src/main.c

```diff
@@ -549,7 +549,7 @@ _Bool ADUC_DeviceClient_Create(ADUC_ConnectionInfo* connInfo, const ADUC_LaunchA
     // This *MUST* be set before the client is connected to IoTHub.  We do not automatically connect when the
     // handle is created, but will implicitly connect to subscribe for device method and device twin callbacks below.
     else if (
-        (iothubResult = IoTHubDeviceClient_LL_SetOption(g_iotHubClientHandle, OPTION_MODEL_ID, g_aduModelId))
+        (iothubResult = ClientHandle_SetOption(g_iotHubClientHandle, OPTION_MODEL_ID, g_aduModelId))
         != IOTHUB_CLIENT_OK)
```

Comment:
Could you check with IoT team whether we need to set a Model ID for Module connection please?
We can leave this as is for now, until we know more.

Reply:
Will do. Worst case I'll make a bug do you know who to reach out to?

---
In reply to: [591980069](https://github.com/Azure/iot-hub-device-update/pull/12#discussion_r591980069) [](ancestors = 591980069)

Reply:
Perhaps John Spaith

---
In reply to: [592828446](https://github.com/Azure/iot-hub-device-update/pull/12#discussion_r592828446) [](ancestors = 592828446,591980069)